x/ccv/provider/types/keys.go

```diff
@@ -28,113 +28,181 @@ const (
 
 	// QuerierRoute is the querier route for IBC transfer
 	QuerierRoute = ModuleName
+)
+
+// Iota generated keys/byte prefixes (as a byte), supports 256 possible values
+const (
+
+	// PortKey defines the key to store the port ID in store
+	PortByteKey byte = iota
 
-	// ChainToChannelKeyPrefix is the key prefix for storing mapping
+	// MaturedUnbondingOpsByteKey is the byte key that stores the list of all unbonding operations ids
+	// that have matured from a consumer chain perspective,
+	// i.e., no longer waiting on the unbonding period to elapse on any consumer chain
+	MaturedUnbondingOpsByteKey
+
+	// ValidatorSetUpdateIdByteKey is the byte key that stores the current validator set update id
+	ValidatorSetUpdateIdByteKey
+
+	// ChainToChannelBytePrefix is the byte prefix for storing mapping
 	// from chainID to the channel ID that is used to send over validator set changes.
-	ChainToChannelKeyPrefix = "chaintochannel"
+	ChainToChannelBytePrefix
 
-	// ChannelToChainKeyPrefix is the key prefix for storing mapping
+	// ChannelToChainBytePrefix is the byte prefix for storing mapping
 	// from the CCV channel ID to the consumer chain ID.
-	ChannelToChainKeyPrefix = "channeltochain"
+	ChannelToChainBytePrefix
 
-	// ChainToClientKeyPrefix is the key prefix for storing the consumer chainID for a given consumer clientid.
-	ChainToClientKeyPrefix = "chaintoclient"
+	// ChainToClientBytePrefix is the byte prefix for storing the consumer chainID for a given consumer clientid.
+	ChainToClientBytePrefix
 
-	// PendingClientKeyPrefix is the key prefix for storing the pending identified consumer chain client before the spawn time occurs.
+	// PendingClientBytePrefix is the byte prefix for storing the pending identified consumer chain client before the spawn time occurs.
```

Comment:
Could you change the name to `PendingCreateProposalBytePrefix` and make it consistent throughout the code.

Reply:
makes sense 👍, addressed in https://github.com/cosmos/interchain-security/pull/242/commits/853150edf824880f16a4e3aa364f665179e08891